<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluginMainWindow</class>
 <widget class="QMainWindow" name="PluginMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Malcore</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icons/images/icon.png</normaloff>:/icons/images/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="buttonUpload">
        <property name="text">
         <string>&amp;Upload</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelModule">
        <property name="text">
         <string>Module:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboModules">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonOptions">
        <property name="text">
         <string>&amp;Options</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="labelStatus">
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Start debugging to analyze a module...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>19</height>
         </size>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextBrowser" name="editReport">
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
      <property name="openLinks">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1030</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionExampleReport"/>
    <addaction name="actionLogin"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <action name="actionExampleReport">
   <property name="text">
    <string>E&amp;xample Report</string>
   </property>
  </action>
  <action name="actionLogin">
   <property name="text">
    <string>&amp;Login</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>buttonUpload</tabstop>
  <tabstop>comboModules</tabstop>
  <tabstop>buttonOptions</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
